packages/cdktf/lib/terraform-element.ts

```diff
@@ -14,11 +14,11 @@ export class TerraformElement extends Construct {
   constructor(scope: Construct, id: string) {
     super(scope, id)
 
-    this.node.addMetadata('stacktrace', 'trace')
+    this.constructNode.addMetadata('stacktrace', 'trace')
     this.stack = TerraformStack.of(this);
   }
 
-  public get node(): Node {
+  public get constructNode(): Node {
```

Comment:
Mmm... Interesting... I am wondering if we should take this into account in constructs 10.x?

Reply:
Potential naming conflicts should be a thought of, yes.